c10/core/SymFloat.cpp

```diff
@@ -62,6 +62,69 @@ SymFloat SymFloat::operator/(const SymFloat& sci) const {
   return SymFloat(res[0]->truediv(res[1]));
 }
 
+SymBool SymFloat::sym_eq(const SymFloat& sci) const {
```

Comment:
Why not just overload the operators in C++ properly since we can do that in C++? Is it because we return a SymBool type?

Reply:
Ah nvm, just saw it in the header.